lib/graph/helpers/ui5Framework.js

```diff
@@ -69,7 +69,7 @@ const utils = {
 		const ui5Dependencies = [];
 		const rootProject = projectGraph.getRoot();
 		await projectGraph.traverseBreadthFirst(async ({project}) => {
-			if (project.isFrameworkProject()) {
```

Comment:
can `project` be `undefined` or `null`?

Reply:
No, this is guaranteed by the `traverseBreadthFirst´ method